src/app/games/epaminondas/EpaminondasTutorial.ts

```diff
@@ -84,17 +74,11 @@ export class EpaminondasTutorial extends Tutorial {
         ),
         TutorialStep.fromPredicate(
             $localize`Moving a phalanx`,
-            $localize`Let us now see how to move multiple pieces along a line, which is called a phalanx:
-        <ol>
-            <li>Click on the first piece of the phalanx.</li>
-            <li>Click on the last piece of the phalanx.</li>
-            <li>Click on one of the squares highlighted in yellow; you can move your phalanx up to a distance equal to its length.</li>
-        </ol><br/>
-        You're playing Dark, Move a phalanx!`,
+            $localize`Let us now see how to move multiple pieces along a line, which is called a phalanx:<ol><li>Click on the first piece of the phalanx.</li><li>Click on one of the squares highlighted in yellow; you can move your phalanx up to a distance equal to its length.</li></ol><br/>You're playing Dark, Move a phalanx!`,
```

Comment:
"multiple pieces, called a phalanx, along a line." je dirais (sinon ça fait "on va bouger plusieurs pièce selon une ligne, [ce mouvement] s'appelle une phalanx")
Point avant le "you can move your phalanx"